csrc/tensor_metadata.cpp

```diff
@@ -340,8 +340,11 @@ std::vector<PolymorphicValue> GetMetaData::evaluate(
       PolymorphicValue(Pointer(input.data_ptr(), tv->dtype()));
   concrete_value["logical_size"] = PolymorphicValue(input.sizes().vec());
   concrete_value["logical_stride"] = PolymorphicValue(input.strides().vec());
-  std::tie(concrete_value["alloc_size"], concrete_value["alloc_stride"]) =
-      inferAndValidateAllocationSizesAndStrides(input, tv, ee);
+  {
```

Comment:
Original sources caused build issue with gcc 9.4.0 - could not create an `std::tuple` with reference types returned by `concrete_value` map. The change in a nutshell replace shorter way of expressing the intent with one that relies explicitly on temp variable.

```
Fuser/csrc/tensor_metadata.cpp:343:72:   required from here
/usr/include/c++/9/tuple:133:42: error: cannot bind non-const lvalue reference of type ‘nvfuser::DynamicType<nvfuser::Containers<std::vector, nvfuser::Struct>, nvfuser::Pointer, nvfuser::Opaque, at::Tensor, std::complex<double>, double, long int, bool>&’ to an rvalue of type ‘nvfuser::DynamicType<nvfuser::Containers<std::vector, nvfuser::Struct>, nvfuser::Pointer, nvfuser::Opaque, at::Tensor, std::complex<double>, double, long int, bool>’
...
Fuser/csrc/dynamic_type.h:210:13: note:   after user-defined conversion: ‘constexpr nvfuser::DynamicType<Containers, Ts>::DynamicType(T) [with T = nvfuser::Struct<nvfuser::DynamicType<nvfuser::Containers<std::vector, nvfuser::Struct>, nvfuser::Pointer, nvfuser::Opaque, at::Tensor, std::complex<double>, double, long int, bool> >; Containers = nvfuser::Containers<std::vector, nvfuser::Struct>; Ts = {nvfuser::Pointer, nvfuser::Opaque, at::Tensor, std::complex<double>, double, long int, bool}]’
```